project-base/docker/php-fpm/Dockerfile

```diff
@@ -134,6 +153,11 @@ RUN composer install --optimize-autoloader --no-interaction --no-progress --no-d
 
 RUN php phing build-deploy-part-1-db-independent clean
 
+
+FROM production as production-project
+
+RUN chmod +x ./deploy/deploy-project.sh && ./deploy/deploy-project.sh merge
+
```

Comment:
This does not need to be present in the version 12, this file does not exist in this version